pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -519,11 +492,11 @@ func (ds *Plugin) openConnection(config *configuration, isReadOnly bool) error {
 			sqlDb.Close()
 		}
 
-		ds.log.Info("Connected to SQL database",
-			"type", config.DatabaseType,
-			"version", version,
-			"read_only", isReadOnly,
-		)
+		ds.log.WithFields(logrus.Fields{
```

Comment:
can you add telemetry constants for those 3 strings?

Reply:
:+1: